models/gitcoin/ethereum/gitcoin_ethereum_schema.yml

```diff
@@ -0,0 +1,141 @@
+version: 2
+
+models:
+  - name: gitcoin_ethereum_contribution_dataset
```

Comment:
Maybe we can name it as gitcoin_ethereum_contribution, a bit simple.

Reply:
I tried to keep the naming identical to the source file.